src/test/java/org/assertj/core/api/Assertions_assertThat_asList.java

```diff
@@ -47,7 +48,7 @@ public void should_pass_list_asserts_on_list_strings_with_asList() {
   public void should_fail_list_asserts_on_non_list_objects_even_with_asList() {
     Object nonList = new Object();
 
-    thrown.expectAssertionError("an instance of:%n <java.util.List>%nbut was instance of:%n <java.lang.Object>");
+    thrown.expectAssertionErrorWithMessageContaining(format("an instance of:%n <java.util.List>%nbut was instance of:%n <java.lang.Object>"));
```

Comment:
https://github.com/joel-costigliola/assertj-core/blob/2.x/src/test/java/org/assertj/core/test/ExpectedException.java#L108 currently does not format the `String` parameters. Should I add string formatting to the method?